go-releaser failed

```
   • releasing using goreleaser 0.112.2...
   • loading config file       file=.goreleaser.yml
   • RUNNING BEFORE HOOKS
   • LOADING ENVIRONMENT VARIABLES
   • GETTING AND VALIDATING GIT STATE
      • releasing v0.0.31, commit d43cdfdf01844f6842f60b1cd690ee2e840ba1c1
   • PARSING TAG      
   • SETTING DEFAULTS 
      • LOADING ENVIRONMENT VARIABLES
      • SNAPSHOTING      
      • GITHUB/GITLAB RELEASES
      • PROJECT NAME     
      • BUILDING BINARIES
      • ARCHIVES         
            • DEPRECATED: `archive` should not be used anymore, check https://goreleaser.com/deprecations#archive for more info.
      • LINUX PACKAGES WITH NFPM
            • DEPRECATED: `nfpm` should not be used anymore, check https://goreleaser.com/deprecations#nfpm for more info.
      • SNAPCRAFT PACKAGES
      • CALCULATING CHECKSUMS
      • SIGNING ARTIFACTS
      • DOCKER IMAGES    
      • ARTIFACTORY      
      • S3               
      • BLOB             
      •0m HOMEBREW TAP FORMULA
         • optimistically guessing `brew[0].installs`, double check
      • SCOOP MANIFEST   
   • SNAPSHOTING      
      • pipe skipped              error=not a snapshot
   • CHECKING ./DIST  
   • WRITING EFFECTIVE CONFIG FILE
      • writing                   config=dist/config.yaml
   • GENERATING CHANGELOG
      • writing                   changelog=dist/CHANGELOG.md
   • BUILDING BINARIES
      • building                  binary=dist/kube-bench_linux_amd64/kube-bench
      • building                  binary=dist/kube-bench_darwin_amd64/kube-bench
   ⨯ release failed after 0.11s error=failed to build for darwin_amd64: cmd/util.go:13:2: cannot find package "github.com/fatih/color" in any of:
	/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/github.com/fatih/color (from $GOROOT)
	/home/travis/gopath/src/github.com/fatih/color (from $GOPATH)
check/check.go:26:2: cannot find package "github.com/golang/glog" in any of:
	/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/github.com/golang/glog (from $GOROOT)
	/home/travis/gopath/src/github.com/golang/glog (from $GOPATH)
cmd/database.go:9:2: cannot find package "github.com/jinzhu/gorm" in any of:
	/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/github.com/jinzhu/gorm (from $GOROOT)
	/home/travis/gopath/src/github.com/jinzhu/gorm (from $GOPATH)
cmd/database.go:10:2: cannot find package "github.com/jinzhu/gorm/dialects/postgres" in any of:
	/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/github.com/jinzhu/gorm/dialects/postgres (from $GOROOT)
	/home/travis/gopath/src/github.com/jinzhu/gorm/dialects/postgres (from $GOPATH)
cmd/federated.go:19:2: cannot find package "github.com/spf13/cobra" in any of:
	/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/github.com/spf13/cobra (from $GOROOT)
	/home/travis/gopath/src/github.com/spf13/cobra (from $GOPATH)
cmd/common.go:27:2: cannot find package "github.com/spf13/viper" in any of:
	/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/github.com/spf13/viper (from $GOROOT)
	/home/travis/gopath/src/github.com/spf13/viper (from $GOPATH)
check/controls.go:21:2: cannot find package "gopkg.in/yaml.v2" in any of:
	/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/gopkg.in/yaml.v2 (from $GOROOT)
	/home/travis/gopath/src/gopkg.in/yaml.v2 (from $GOPATH)
check/test.go:27:2: cannot find package "k8s.io/client-go/util/jsonpath" in any of:
	/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/k8s.io/client-go/util/jsonpath (from $GOROOT)
	/home/travis/gopath/src/k8s.io/client-go/util/jsonpath (from $GOPATH)
```

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -2,7 +2,6 @@ builds:
   - main: main.go
     binary: kube-bench
     goos:
-      - darwin
       - linux
     goarch:
       - amd64

